modules/apps/object/object-test/src/testFunctional/macros/ObjectNotifications.macro

```diff
@@ -241,70 +241,100 @@ definition {
 			key_name = ${name},
 			locator1 = "ObjectAdmin#TABLE_LIST_TITLE");
 
-		AssertTextEquals(
-			key_text = "Name",
-			locator1 = "TextInput#ANY",
-			value1 = ${name});
-
-		AssertTextEquals(
-			key_text = "From Address",
-			locator1 = "TextInput#ANY",
-			value1 = ${fromAddress});
-
-		AssertTextEquals(
-			key_text = "From Name",
-			locator1 = "TextInput#ANY",
-			value1 = ${fromName});
-
-		if (isSet(description)) {
-			AssertTextEquals(
-				key_text = "description",
-				locator1 = "TextArea#ANY",
-				value1 = ${description});
-		}
-
-		if (isSet(entryTo)) {
+		if (${notificationType} == "Email") {
 			AssertTextEquals(
-				key_text = "To",
+				key_text = "Name",
 				locator1 = "TextInput#ANY",
-				value1 = ${entryTo});
-		}
+				value1 = ${name});
 
-		if (isSet(cc)) {
 			AssertTextEquals(
-				key_text = "CC",
+				key_text = "From Address",
 				locator1 = "TextInput#ANY",
-				value1 = ${cc});
-		}
+				value1 = ${fromAddress});
 
-		if (isSet(bcc)) {
 			AssertTextEquals(
-				key_text = "BCC",
+				key_text = "From Name",
 				locator1 = "TextInput#ANY",
-				value1 = ${bcc});
-		}
+				value1 = ${fromName});
 
-		if (isSet(subject)) {
+			if (isSet(description)) {
+				AssertTextEquals(
+					key_text = "description",
+					locator1 = "TextArea#ANY",
+					value1 = ${description});
+			}
+
+			if (isSet(entryTo)) {
```

Comment:
Remember to sort stuff like this alphabetically, unless we need to add them in a specific order